package errors_test

func oops() error
